src/service/globals/mod.rs

```diff
@@ -188,7 +190,11 @@ impl Service {
             RoomVersionId::V5,
             RoomVersionId::V11,
         ];
-
+        let argon = Argon2::new(
+            argon2::Algorithm::Argon2id,
+            argon2::Version::default(),
+            argon2::Params::new(2432, 2, 1, None).expect("valid parameters"),
```

Comment:
Can we add a comment what these config parameters are? I'm guessing `2432` is kilobytes, `2` is iterations, and `1` is amount of parallelism.

I'd like to use the following from https://cheatsheetseries.owasp.org/cheatsheets/Password_Storage_Cheat_Sheet.html#argon2id

> m=19456 (19 MiB), t=2, p=1 (Do not use with Argon2i)

Reply:
this is the one actually, I should add a comment yeah

Reply:
Are you sure that's the one? 2432 kilobytes is ~2.5mb, that isn't 19MiB.

Reply:
Kib != Kb

Reply:
https://docs.rs/argon2/latest/argon2/struct.Params.html#arguments

Reply:
It's still not 19MiB.

> m_cost: memory size in 1 KiB blocks. Between 1 and (2^32)-1.

<img width="504" alt="image" src="https://github.com/girlbossceo/conduwuit/assets/112147643/7ca2bc29-530d-4a03-8f52-c03dad91414d">

Reply:
sorry I fixed it, motherfucking wolfram got high and misinterpreted Mib as MebiBits